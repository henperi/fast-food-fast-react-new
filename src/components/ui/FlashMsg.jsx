import React from 'react';
// import { toastr as flashToast } from 'react-redux-toastr';
// import 'react-redux-toastr/lib/css/react-redux-toastr.min.css';

const FlashMessage = ({ flashData }) => {
  const {
    showFlash, title, message, flashType,
  } = flashData;
  // return <div>{showFlash && <span>{flashToast[flashType](title, message)}</span>}</div>;
  return (
    <div>
      {showFlash && <span className={`card card-shadow flash ${flashType}`}>{message}</span>}
    </div>
  );
};

export default FlashMessage;
