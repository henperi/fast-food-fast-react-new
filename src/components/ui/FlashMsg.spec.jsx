import React from 'react';

/* eslint-disable no-undef */
import FlashMsg from './FlashMsg';

describe('The Errors component Test Suite', () => {
  const props = {
    flashData: {
      showFlash: '',
      title: '',
      message: '',
      flashType: '',
    },
  };
  it('should return a main error list item', () => {
    const wrapper = shallow(<FlashMsg {...props} />);

    expect(wrapper.length).toBe(1);
    // expect(wrapper).toMatchSnapshot();
    expect(wrapper.find(FlashMsg)).toBeDefined();
  });

  // it('should format a date and return it to standard callendar dates', () => {
  //   const wrapper = shallow(<Errors errors={[{ param: 'email' }]} name="email" />);

  //   expect(wrapper.length).toBe(1);
  //   // expect(wrapper).toMatchSnapshot();
  //   expect(wrapper.find(FlashMsg)).toBeDefined();
  // });
});
