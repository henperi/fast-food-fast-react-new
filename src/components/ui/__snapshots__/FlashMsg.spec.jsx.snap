// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Errors component Test Suite should return a main error list item 1`] = `<div />`;
